lib/ContainerManager.js

```diff
@@ -200,15 +207,19 @@ Server.prototype.runBuild = function* (build, log) {
     image: image,
     imageConfig: self.config.images[image],
 
-    // global config
+    // Global configuration for what binds to expose to all containers.
     binds: self.config.binds,
     // attachLogs: true, // set to true to attach 'docker -f logs' to stdout of this process
 
-    assetsUrl: this.config.assets.url
+    // TODO: Can we have a more generic way of passing plugin config through
+    // without explicit knowledge of plugin configuration here?
+    assetsUrl: this.config.assets.url,
+    // Attach our our logger to the request options.
+    log: log,
+
   };
 
-  log.info(`Starting container build: ${containerName}`)
-  // log.debug({build, container_config}, "Starting container build")
+  log.info({build, container_config}, `Starting container build: ${containerName}`)
```

Comment:
I had this commented out because it'd get too noisy dumping the whole build. I'd keep a short `.info` log, and a more complete `.debug` log.